xmppserver/src/main/webapp/session-summary.jsp

```diff
@@ -303,7 +269,22 @@
         </c:if>
         <th nowrap colspan="2"><fmt:message key="session.details.status" /></th>
         <th nowrap colspan="2"><fmt:message key="session.details.presence" /></th>
-        <th nowrap><fmt:message key="session.details.priority" /></th>
+        <th nowrap>
+            <a href="session-summary.jsp" onclick='return toggleColumnOrder(${SessionResultFilter.SORT_NUM_CLIENT_PACKETS})'>
+                <fmt:message key="session.details.received-abbreviation" />
+                <c:if test="${listPager.sortColumnNumber == SessionResultFilter.SORT_NUM_CLIENT_PACKETS}">
+                    <img src="images/sort_${listPager.sortDescending ? "descending" : "ascending"}.gif" alt="The sort order in this column is ${listPager.sortDescending ? "descending" : "ascending"} (click to toggle).">
+                </c:if>
+            </a>
+        </th>
+        <th nowrap>
+            <a href="session-summary.jsp" onclick='return toggleColumnOrder(${SessionResultFilter.SORT_NUM_SERVER_PACKETS})'>
+                <fmt:message key="session.details.transmitted-abbreviation" />
```

Comment:
Should (can?) this have the unabbreviated text as a hover/tooltip?

Reply:
I've pushed a commit that does it, by adding a title attribute to the `th`, but I didn't have the right strings, so did a _tiny_ refactor to make everything tidy. Feel free to bin it or amend it as you like - this tweak is almost as big as the change that it tweaks :)

Reply:
Just tested it: works for me. I'm happy to have that go in.